docs/hosting/additional-concepts.md

```diff
@@ -0,0 +1,153 @@
+```python exec
+import reflex as rx
+from reflex_image_zoom import image_zoom
+from pcweb.pages import docs
+from pcweb.styles.styles import get_code_style, cell_style
+
+
+regions = {
+    "ams": "Amsterdam, Netherlands",
+    "arn": "Stockholm, Sweden",
+    "alt": "Atlanta, Georgia (US)",
+    "bog": "Bogotá, Colombia",
+    "bom": "Mumbai, India",
+    "bos": "Boston, Massachusetts (US)",
+    "cdg": "Paris, France",
+    "den": "Denver, Colorado (US)",
+    "dfw": "Dallas, Texas (US)",
+    "ewr": "Secaucus, NJ (US)",
+    "eze": "Ezeiza, Argentina",
+    "fra": "Frankfurt, Germany",
+    "gdl": "Guadalajara, Mexico",
+    "gig": "Rio de Janeiro, Brazil",
+    "gru": "Sao Paulo, Brazil",
+    "hkg": "Hong Kong, Hong Kong",
+    "iad": "Ashburn, Virginia (US)",
+    "jnb": "Johannesburg, South Africa",
+    "lax": "Los Angeles, California (US)",
+    "lhr": "London, United Kingdom",
+    "mad": "Madrid, Spain",
+    "mia": "Miami, Florida (US)",
+    "nrt": "Tokyo, Japan",
+    "ord": "Chicago, Illinois (US)",
+    "otp": "Bucharest, Romania",
+    "phx": "Phoenix, Arizona (US)",
+    "qro": "Querétaro, Mexico",
+    "scl": "Santiago, Chile",
+    "sea": "Seattle, Washington (US)",
+    "sin": "Singapore, Singapore",
+    "sjc": "San Jose, California (US)",
+    "syd": "Sydney, Australia",
+    "waw": "Warsaw, Poland",
+    "yul": "Montreal, Canada",
+    "yyz": "Toronto, Canada"
+}
+```
+
+
+# Reflex Cloud - Additional Concepts
+
+To go back, i.e. from an app to a project or from a project to your list of projects you just click the `REFLEX logo` in the top left corner of the page.
+
+## Environment Variables
+
+Below is an example of how to use an environment variable in a deployment command:
+
+```bash
+reflex deploy --project f88b1574-f101-####-####-5f########## --env OPENAI_API_KEY=sk-proj-vD4i9t6U############################
+```
+
+They are passed after the `--env` flag as key value pairs. 
+
+To pass multiple environment variables, you can repeat the `--env` tag. i.e. `reflex deploy --project f88b1574-f101-####-####-5f########## --env KEY1=VALUE1 --env KEY2=VALUE`
+
+
+Alternatively if there is an environment variable file in your project directory you can pass the `--envfile` flag with the path to the env file. For example:
```

Comment:
maybe this should be the first option and the "alternate" option is the --env flag?

Reply:
changed the order